test/test_dynamic_shapes.py

```diff
@@ -1772,31 +1772,55 @@ def test_dim_constraints_solve_full(self):
             "dynamic_dim(L['x9'], 1) == dynamic_dim(L['x6'], 1)",
         })
 
-        def dummy_f(x0, x1, x2, x3, x4, x5, x6, x7, x8, x9, x11, x12):
+        def dummy_f(x0, x1, x2, x3, x4, x5, x6, x7, x8, x9, x10, x11, x12):
             pass
 
         action_code = dim_constraints.prettify_results(inspect.signature(dummy_f))
         static_code, dynamic_code = re.findall(r"```(.*?)```", action_code, re.DOTALL)
         print(static_code)
         expected_static = '''
-def specializations(x0, x1, x2, x3, x4, x5, x6, x7, x8, x9, x11, x12):
+def specializations(x0, x1, x2, x3, x4, x5, x6, x7, x8, x9, x10, x11, x12):
+    # x0:
```

Comment:
Could you add a test that would have multiple specializations on x0 to check the grouping? I would expect it to look something like:
```
# x0:
assert x0.size()[0] == <something>
assert x0.size()[1] == <something>
# x1:
assert x1.size()[0] == <something>
assert x1.size()[2] == <something>
```

Reply:
Done!